docs/build-the-docs.md

```diff
@@ -66,7 +67,7 @@ antora --clean --pull --quiet --silent \
 The playbook file (`site.yml`) sets the `site.url` configuration directive to `http://localhost:5000`.
 It's likely fair to assume that this isn't the domain where the documentation will be hosted.
 
-Given that, after the documentation has been generated the search index file (`public/search_index.json`) needs to be updated to change `http://localhost:5000` to the hosting server where the documentation is hosted.
+Given that, after the documentation has been generated, the search index file (`public/search_index.json`) needs to be updated to change `http://localhost:5000` to the hosting server where the documentation is hosted.
```

Comment:
The search doesn't get generated separately. It's a single Antora execution that generates it.

Reply:
I just added a comma for better readability.
Please advice what to write instead

Reply:
Seems like this paragraph needs to be removed.